templates/default/sentry.conf.8.0.py.erb

```diff
@@ -0,0 +1,205 @@
+# This file is just Python, with a touch of Django which means
```

Comment:
There are quite a few moved/unnecessary values held over from < 8.0, but the main difference is between this template the default template that the 'sentry init' command generates through a manual install. A lot of stuff is moved and/or renamed---so much so that it was confusing for me when I was trying to do the upgrade.
Separating the file remedies the discrepancy with sentry's preferred config layout and (I think) makes it way easier for users to quickly tell which template they should use (and how) for their version